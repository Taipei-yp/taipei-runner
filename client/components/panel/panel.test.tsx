import React from "react";
import TestRenderer from "react-test-renderer";
import { Panel } from ".";

describe("Component Panel", () => {
  const panel = TestRenderer.create(
    <Panel className="testclass">
      <div>ChildrenComponent</div>
    </Panel>,
  );

  it("correct render", () => {
    expect(panel.toJSON()).toMatchSnapshot();
  });

  it("correct props", () => {
    expect(panel.root.props.className).toBe("testclass");
  });
});
