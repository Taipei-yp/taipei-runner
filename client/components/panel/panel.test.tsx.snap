// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Panel correct render 1`] = `
<div
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
>
  <div>
    ChildrenComponent
  </div>
</div>
`;
